### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by przemekjuzwiszin**
I see no mistakes. Maybe in the article content. I'm bad at grammatic. Code HTML looks alright.

### CODE REVIEW

File: formularze.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                różnym typom można wyróżnić, np checklisty, radio listy wybór jednej z wielu dostępnych opcji.
+                Kolejnym krokiem w tworzeniu formularza jest walidacja danych. Użytkownik będzie mógł wysłać
+                formularz dopiero po wypełnieniu pól poprawnie według wzoru. Pola formularza są wyposażone we wzory
+                nazywane wyrażeniami nieregularnymi. HTML pozwala nam na umieszczenie w polach formularza walidacji
+                za pomocą samego kodu HTML. Należy jednak pamiętać że walidacja musi być zdefiniowana po stronie
+                backendu. Walidacja i ogólnie formularz może mieć dodaną właściwość ‘required’. Required wymaga od
+                użytkownika wpisanie w dane pole informacji aby wysłać formularz i przejść do dalszej części strony
+                internetowej.
+            </p>
+            <footer class="article__footer">
```
Comment (karbowskam) [inline formularze.html:47]: I thought that a website has only one footer. But I can be wrong :) Could you check that? And I like your 'typo': owrite = alright :)

Comment (przemekjuzwiszin) [reply formularze.html:47]: We can use two footers when one footer is outside the main tag and this footer is the same for all pages and the home page.

Comment (przemekjuzwiszin) [reply formularze.html:47]: Thanks for the typo owrite.  I corrected this funny word. Bravo:)

Comment (karbowskam) [reply formularze.html:47]: Ok. Thanks for your reply :)

**Review by karbowskam — APPROVED** [review]

**Review by robson60000 — APPROVED** [review]

**Pull request merged**